modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/NamedSettings.kt

```diff
@@ -57,6 +57,7 @@ data class ChangedNamedSettings(
     @field:Json(name = "autoShowPlayed") val autoShowPlayed: NamedChangedSettingBool? = null,
     @field:Json(name = "autoPlayLastListUuid") val autoPlayLastSource: NamedChangedSettingString? = null,
     @field:Json(name = "useEmbeddedArtwork") val useEmbeddedArtwork: NamedChangedSettingBool? = null,
+    @field:Json(name = "useEpisodeArtowork") val useEpisodeArtwork: NamedChangedSettingBool? = null,
```

Comment:
The setting will be synced at some point but this needs server changes first before we can support on the client side so its better to remove it for now.